docs/basic-docs/atomicdex-api-20-dev/get_locked_amount.md

```diff
@@ -0,0 +1,84 @@
+# get\_locked\_amount
+
+The `get_locked_amount` method returns the amount of a coin which is currently locked by a swap which is in progress. If the coin queried is not activated, a `NoSuchCoin` error will be returned.
```

Comment:
2nd sentence can be shorter a bit: "If the coin is not activated, a `NoSuchCoin` error will be returned."